tensorboard/plugins/graph/tf_graph_loader/tf-graph-dashboard-loader.ts

```diff
@@ -151,7 +155,7 @@ class TfGraphDashboardLoader extends LegacyElementMixin(PolymerElement) {
         const params = new URLSearchParams();
         params.set('tag', tag!);
         params.set('run', run);
-        const metadataPath = getRouter().pluginRoute(
+        const metadataPath = getRouter().pluginRouteForSrc(
```

Comment:
This seems safe but, FWIW, this is the only code path I couldn't personally test. I don't know how to trigger a "Profile" graph type.

Reply:
The `graphs_demo` generates a profile (see [here](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/graph/graphs_demo.py#L106)), however I can't get any of them to load (seemingly related to #1961).

This should be safe, but I can revert this one in particular if you'd prefer (it's not directly relevant to our internal needs).

Reply:
I'm ok converting it to pluginRouteForSrc.